src/components/SignupsSplit.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ButtonLink from './ButtonLink';
+import ButtonGroup from './ButtonGroup';
+import { Link } from 'react-router-dom';
+import { useTreatments, SplitContext } from '@splitsoftware/splitio-react';
+
+const SignupsSplit = (props) => {
+  const treatments = useTreatments(['signups']);
+  const treatment = treatments['singups'];
```

Comment:
looks like this may be spelled wrong should it be `signups`? Also as per the linter we can change it to dot notation
```suggestion
  const treatment = treatments.signups;
```

Reply:
When I talked to @aswanson-nr he suggested to leave it as `'signups'`, since we're referencing the string. To avoid having to create a const for the string just to use it here.

Reply:
hmm maybe, i'm not actually sure what format the `treatments` data is in so Alec might be right, though in that case we'll need to add an exclusion rule to ignore this linting error